js/views/adScreen.js

```diff
@@ -187,6 +202,13 @@ var AdScreen = React.createClass({
           {playbackControlItems}
         </div>
 
+        {showUnmute ? <button className="oo-ad-screen oo-unmute"
+          onClick={this.unmuteClick}
```

Comment:
Can we add `type="button"`? If we want this to be accessible this will also need `tabindex=0`

Reply:
Done.